worker-specific-task-queues/workflow.go

```diff
@@ -11,6 +11,18 @@ import (
 // FileProcessingWorkflow is a workflow that uses Worker-specific Task Queues to run multiple Activities on a consistent
 // host.
 func FileProcessingWorkflow(ctx workflow.Context) (err error) {
+	for attempt := 1; attempt <= 5; attempt++ {
+		if err = processFile(ctx); err == nil {
```

Comment:
Arguably one might only retry on schedule to close timeout, but this is probably fine too.